molecule/check_facts/tasks/prepare_to_check.yml

```diff
@@ -0,0 +1,5 @@
+---
+
+- set_fact:
```

Comment:
`prepare_to_check.yml` name confuses - I was thinking about molecule prepare.
What about `set_delivered_package_path_step`?

Reply:
But it's possible to make some other tasks here